client/me/application-passwords/index.jsx

```diff
@@ -193,7 +193,7 @@ class ApplicationPasswords extends Component {
 			<Fragment>
 				<QueryApplicationPasswords />
 
-				<SectionHeader label={ translate( 'Application Passwords' ) }>
+				<SectionHeader label={ translate( 'Application passwords' ) }>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.